content/docs/command-reference/exp/init.md

```diff
@@ -128,11 +128,9 @@ $ dvc exp init './another_script.sh $MYENVVAR'
   your experiment can be found (if any). Overrides other configuration and
   default value (`plots/`).
 
-- `--live` - configure the `path` directory for [DVCLive](/doc/dvclive). This is
-  where experiment logs will be written. Overrides other configuration and
-  default value (`dvclive/`).
-
-  > This only has an effect when used with `--type=dl`.
+- `--live` - use the given path to set up `metrics` and `plots` as in
```

Comment:
`--type=dl` is no longer needed? Is it still a thing? That option desc. may also need an update.

Reply:
The thing is that:

> This only has an effect when used with `--type=dl`

Was not true at all. Passing `live` can have effect regardless of whether you use `--type=dl` or not

Reply:
OK. So is the `--type` description accurate?

> dl stages are intended for use in deep-learning scenarios, where metrics and plots are tracked with [DVCLive](https://dvc.org/doc/dvclive).

Reply:
> OK. So is the --type description accurate? 

It needs to be revisited now that https://github.com/iterative/dvc/pull/7703 has been merged. It belongs to different P.R. though

Reply:
Commented in https://github.com/iterative/dvc/pull/7703#pullrequestreview-969077209